### PR TITLE
First version

### PR DESCRIPTION
**Pull request by patrick-scherer**

### CODE REVIEW

File: frontend/selectors/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { createSelector } from 'reselect';
```
Comment (patrick-scherer) [inline frontend/selectors/index.js]: will delete this file...

File: frontend/components/CartNotice/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * The CartNotice component
+ */
+const CartNotice = ({ isOrderable }) => (
+  !isOrderable ? (
+    <div className={styles.notice}>
+      <I18n.Text string="minimalOrderValue.noticeText">
+        <strong forKey="mov">
+            <I18n.Price price={minimalOrderValue} currency="EUR" />
```
Comment (Carsten89) [inline frontend/components/CartNotice/index.jsx]: you should use the currency of the cart here

File: frontend/components/CartNotice/index.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </strong>
+      </I18n.Text>
+    </div>
+  ): null
+);
+
+CartNotice.propTypes = {
+  points: PropTypes.bool,
```
Comment (Carsten89) [inline frontend/components/CartNotice/index.jsx]:
```suggestion
  isOrderable: PropTypes.bool,
```
;)

File: frontend/components/CartNotice/style.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import { themeConfig } from '@shopgate/pwa-common/helpers/config';
+
+const { colors } = themeConfig;
+
+const notice = css({
+  backgroundColor: colors.warning,
+  padding: 16,
+  width: '100%',
+  display: 'inline-block',
+  color: '#fff',
```
Comment (Carsten89) [inline frontend/components/CartNotice/style.js]: try to use a color variable also here.

File: frontend/locale/de-DE.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "minimalOrderValue": {
+    "noticeText": "Der Mindestbestellwert von {mov} wurde noch nicht erreicht."
```
Comment (Carsten89) [inline frontend/locale/de-DE.json:3]: we should have at least also an en-US file

File: frontend/components/CartNotice/index.jsx (hunk 3 above)
Comment (patrick-scherer) [reply frontend/components/CartNotice/index.jsx]: :D

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**